Server/Plugins/APIDump/InfoFile.html

```diff
@@ -150,6 +156,8 @@
 
 		<p>The permission element specifies that the command is only available with the specified permission. Note that the permission for subcommand's parent isn't checked when the subcommand is called. This means that specifying the permission for a command that has subcommands has no effect whatsoever, but is discouraged because we may add processing for that in the future.</p>
 
+		<p>The Category string is only used for generating the documentation. Specifying different categories will result in commands grouped together, not specifying a category will place that command in the "General" category. Categories are optional.</p>
```

Comment:
Perhaps you can `<a href="#Categories">link this to the Categories section</a>`?